model/src/main/kotlin/AdvisorRun.kt

```diff
@@ -31,12 +31,12 @@ data class AdvisorRun(
     /**
      * The [Instant] the advisor was started.
      */
-    val startTime: Instant,
+    val startTime: Instant = Instant.EPOCH,
```

Comment:
> To align with the other *Run classes.

Are you aware the in the other *Run classes the comment says "The default value exists only for backward compatibility"? So IMO we do not need / want this here.